nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -6,7 +6,7 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 ## Highlights {#sec-release-21.11-highlights}
 
-- Nix has been updated to version 2.4, reference its [release notes](https://discourse.nixos.org/t/nix-2-4-released/15822) for more information on what has changed. The previous version of Nix, 2.3.16, remains available for the time being in the `nix_2_3` package.
+- Nix version remains at 2.3.16. Nix has not been updated to version 2.4 due to regressions in non-experimental behavior. To upgrade to 2.4, use `nixos-unstable` branch or `nixUnstable`/`nix_2_4` attributes; reference the [release notes](https://discourse.nixos.org/t/nix-2-4-released/15822) for more information on upcoming changes. Provide breakage reports at [https://github.com/NixOS/nix/issues]()
```

Comment:
```suggestion
- Nix version remains at 2.3.16. Nix has not been updated to version 2.4 due to regressions in non-experimental behavior. To upgrade to 2.4, use the `nixos-unstable` branch or set the `nix.package` options to either the `nixUnstable` or `nix_2_4` attributes. Read the [release notes](https://discourse.nixos.org/t/nix-2-4-released/15822) for more information on upcoming changes. Please help us improving Nix by providing any breakage reports at [https://github.com/NixOS/nix/issues]()
```

Don't take the whole rewrite as the only way to change this. Main thing to consider is making better structured sentences, *and importantly* to tell which option to set to use Nix 2.4.